bedrock/firefox/views.py

```diff
@@ -492,7 +493,7 @@ def get_template_names(self):
                 if locale == "en-GB" or country == "GB":
                     template = "firefox/whatsnew/whatsnew-fx117-uk-reader-view.html"
                 else:
-                    template = "firefox/whatsnew/index.html"
+                    template = "firefox/whatsnew/whatsnew-fx117-na-relay.html"
```

Comment:
I think we should move this above the UK reader page in a rule like:

```
if locale in ["en-US", "en-CA"]:
    template = "firefox/whatsnew/whatsnew-fx117-na-relay.html"
elif locale == "en-GB" or country == "GB":
    template = "firefox/whatsnew/whatsnew-fx117-uk-reader-view.html"
else:
    template = "firefox/whatsnew/index.html"
```

That keeps the default as the fallback for other locales. This whole rule will be changed around when the VPN page gets added but I think we should make sure it's ready to go live without it, just in case.